src/Our.Umbraco.Ditto/ComponentModel/Attributes/UmbracoPropertyProcessorAttribute.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace Our.Umbraco.Ditto
+{
+    /// <summary>
+    /// The Umbraco property processor attribute.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property)]
+    public class UmbracoPropertyProcessorAttribute : DittoProcessorAttribute
```

Comment:
As discussed, I'd like to remove the "Processor" suffix from the established attributes.

I understand having the suffix would make it more explicit, but I feel that the usage for a developer would be, "I want to get an Umbraco property value", not that they want to initiate a process pipeline.

Does that sound reasonable?